iosApp/iosAppTests/Views/ContentViewTests.swift

```diff
@@ -150,12 +150,12 @@ final class ContentViewTests: XCTestCase {
         wait(for: [hasAppeared, loadConfigCallback], timeout: 5)
     }
 
-    @MainActor func testHidesMap() throws {
+    @MainActor func testDoesNotHideMap() throws {
```

Comment:
I thought about removing or skipping this test, but inverting it instead will make it easier to remember to switch the test back when we restore the functionality.